lib/dpla/conditions.rb

```diff
@@ -11,13 +11,19 @@ def to_s
     private
 
     def transform(conditions)
+      # When is `conditions' not an array or hash?  It seems that it should
```

Comment:
This looks right, but I think it's good practice to use exhaustive `case` statements. Falling back on `encode_uri(conditions)` seems pretty reasonable.